examples/jss/src/index.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import { render } from 'react-dom';
 
-import './global-styles';
+import GlobalStyles from './components/global-styles';
 import App from './components/app';
 
-render(<App />, document.getElementById('example-root'));
+render(<GlobalStyles><App /></GlobalStyles>, document.getElementById('example-root'));
```

Comment:
Yeah I get you, but this still isn't as nice as CSS modules, where all global selectors (`html`, `body`,…) are just that, global. (and you can opt-in to global selectors specifically with `:global`, though I'm not too big of a fan of that since you shouldn't have a use case for it) Still inclined to give this a "Kinda"…